pkg/graph/schema.graphql

```diff
@@ -421,7 +421,10 @@ union TaggedEntity =  UserAccount | PossibleOperationalSolution
 TaggedContent represents content that has a tag in it. It is composed of the raw tag text, as well as the array of possible tags
 """
 type TaggedHTML {
-  rawContent: String! #TODO: SW change this to an HTML type?
+  """
+  RawContent is HTML. It is sanitized on the backend
+  """
+  rawContent: String! 

```

Comment:
I did some exploratory work at making an HTML scalar, but it felt convoluted. Instead, TaggedHTMLInput is a scalar which sanitizes the html, and leaves the front end representation as a string.